packages/freezed/test/json_test.dart

```diff
@@ -133,6 +133,130 @@ Future<void> main() async {
         Generic(42),
       );
     });
+
+    group('with argument factories', () {
```

Comment:
Make sure to specify the generic types inside tests instead of relying on type inference

This ensures that fromJson wouldn't be deserializing to Object/dynamic somehow

Reply:
Done